src/lib.rs

```diff
@@ -226,9 +226,9 @@ pub use source_futures::*;
 // This works around it by using our own counter for threads.
 //
 // Taken from the fragile crate
-use std::sync::atomic::{AtomicUsize, Ordering, ATOMIC_USIZE_INIT};
+use std::sync::atomic::{AtomicUsize, Ordering};
 fn next_thread_id() -> usize {
-    static mut COUNTER: AtomicUsize = ATOMIC_USIZE_INIT;
+    static mut COUNTER: AtomicUsize = AtomicUsize::new(0);
```

Comment:
Ah is this stabilized since 1.31 already (`const`ified constructor)? I thought it was later :)

:+1: if it builds with 1.31